prime-router/docs/getting-started/postman/README.md

```diff
@@ -0,0 +1 @@
+# Postman
```

Comment:
@JessicaWNava what do we think about renaming this to swagger and just not documenting postman at all? Swagger can serve the same purpose as postman, don't see why we need both? Anyone else have thoughts?

Reply:
Can you have store collections with example requests with Swagger like we do in Postman?